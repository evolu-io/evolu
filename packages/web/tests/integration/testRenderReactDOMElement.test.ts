import { pageUrl } from './helpers/pageUrl';
import { pageDom } from './helpers/pageDom';

beforeEach(async () => {
  await page.goto(pageUrl('testRenderReactDOMElement'));
});

test('render', async () => {
  await expect(await pageDom()).toMatchSnapshot();
});
