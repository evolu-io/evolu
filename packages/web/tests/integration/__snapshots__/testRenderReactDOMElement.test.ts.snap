// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div
  id="__next"
>
  <div
    autocorrect="off"
    contenteditable="true"
    data-gramm="true"
    role="textbox"
    spellcheck="false"
    tabindex="0"
  >
    <div>
      <div
        style="font-size:24px"
      >
        ·heading·
      </div>
      <div
        style="font-size:16px"
      >
        <br />
      </div>
      <img
        alt="Square placeholder image 80px"
        height="80"
        src="https://via.placeholder.com/80"
        width="80"
      />
    </div>
  </div>
</div>
`;
